src/encoded/static/components/antibody.js

```diff
@@ -62,23 +62,26 @@ const LotComponent = (props, reactContext) => {
     const organismComponents = [];
     const organismTerms = [];
     const organismTips = [];
-    const geneComponents = [];
-    const geneTerms = [];
-    const geneTips = [];
+    let geneName = [];
+    let geneComponents = [];
+    let geneTerms = [];
     targetKeys.forEach((key, i) => {
         const scientificName = targets[key].organism.scientific_name;
-        const geneName = targets[key].gene_name;
+        if (targets[key].genes) {
+            geneName = targets[key].genes.map(gene => gene.symbol);
+        }
 
         // Add to the information on organisms from the targets
         organismComponents.push(<span key={key}>{i > 0 ? <span> + <i>{scientificName}</i></span> : <i>{scientificName}</i>}</span>);
         organismTerms.push(`targets.organism.scientific_name=${scientificName}`);
         organismTips.push(scientificName);
 
         // Add to the information on gene names from the targets
-        if (geneName && geneName !== 'unknown') {
-            geneComponents.push(<span key={key}>{i > 0 ? <span> + {geneName}</span> : <span>{geneName}</span>}</span>);
-            geneTerms.push(`targets.gene_name=${geneName}`);
-            geneTips.push(geneName);
+        if (geneName.length) {
```

Comment:
Very minor, but the AirBnB style guide specifies numeric comparisons, even against 0, must be explicit https://github.com/airbnb/javascript#comparison--shortcuts

Reply:
Thank you for pointing this out. Very good practice to learn.